modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -176,164 +169,165 @@ $ ls $HOME/clusterconfig/
-including `bootstrap.ign`, `master.ign` and `worker.ign`, to the carry out the
-installation. It is not recommended to modify these files.
+such as `bootstrap.ign`, `master.ign` and `worker.ign`, to carry out the
+installation.
++
+[IMPORTANT]
+====
+It is not recommended to modify these files.
+====
 +
-For PXE installs, you pass the Ignition configs on the APPEND line using the
+For PXE installations, you pass the Ignition configs on the `APPEND` line using the
 `coreos.inst.ignition_url=` option. For ISO installs, after the ISO boots to
 the shell prompt, you identify the Ignition config on the `coreos-installer`
 command line with the `--ignition-url=` option.
 +
-Instead of passing the location of an Ignition config via a kernel or
-command-line option, you can embed an Ignition config into the ISO
-installer image. This allows you to do bare metal installs with the ISO,
-without requiring access to an HTTP server. See “Embedding an Ignition
-config in the {op-system-first} ISO” for details.
+For ISO installations, instead of passing the location of an Ignition config using a kernel or
+command-line option, you can embed an Ignition config into the ISO installer image. This allows you to do bare metal installs with the ISO, but without requiring access to an HTTP server. See “Embedding an Ignition config in the {op-system-first} ISO” for details.
 
-[id="installation-user-infra-machines-advanced_embedignition_{context}"]
-=== Embedding an Ignition config in the {op-system-first} ISO
-The following procedures describe how to embed an Ignition config into
-the ISO so it is applied when the new installation first boots from disk.
+* **Live install Ignition config**: This type must be created manually and should be avoided if possible, as it is not supported by Red Hat. With this method, the Ignition config passes to the live install medium, runs immediately upon booting, and performs setup tasks before and/or after the {op-system} system installs to disk. This method should only be used for performing tasks that must be performed once and not applied again later, such as with advanced partitioning that cannot be done using a MachineConfig.
++
+For PXE or ISO boots, you can create the Ignition config
+and `APPEND` the `ignition.config.url=` option to identify the location of
+the Ignition config. You also need to append `ignition.firstboot ignition.platform.id=metal`
+or the `ignition.config.url` option will be ignored.
 
-To embed an Ignition config named `worker.ign` into an ISO image
-(for example rhcos-<version>-live.x86_64.iso), copy the image to
-a local directory, then run the coreos-installer container with
-that directory mounted, as follows:
+[id="installation-user-infra-machines-advanced_embedignition_{context}"]
+=== Embedding an Ignition config in the {op-system} ISO
+You can embed an Ignition config at the shell prompt, and then identify it on the `coreos-installer` command line.
```

Comment:
I'm not sure what "then identify it on the `coreos-installer` command line" means.

```suggestion
You can embed a live install Ignition config directly in an {op-system} ISO image. When
the ISO image is booted, the embedded config will be applied automatically.
```

Reply:
Resolved. (FWIW, the previous text came from a line in Chris's doc: "For ISO installs, after the ISO boots to the shell prompt, you identify the Ignition config on the coreos-installer command line with the --ignition-url= option.")

Reply:
That's not describing the embedding case, though, and also it gives specifics on how to perform the suggested operation.

Text looks good in the current draft.